File: refactor idea in Java/CLIView.java
import java.util.List;

public class CLIView implements IPlayersView {

    @Override
    public void display(List<Player> players) {
        for (Player player : players) {
            System.out.println(player.name);
        }
    }


    // non-static display

    // CLIView view1 = new CLIView(), view2 = new CLIView();
    // view1.display() != view2.display()

    // static display
    // CLIView view1 = new CLIView(), view2 = new CLIView();
    // wrong view1.display()
    // CLIView.display()
}

File: PHP_implement/View/CLIView.php
<?php

require_once "IPlayersView.php";

class CLIView implements IPlayersView{
    function display($playersCollection){
        $players = $playersCollection->toArray();
        echo "Current Players: \n";
        foreach ($players as $player) {
            echo "\tName: $player->name\n";
            echo "\tAge: $player->age\n";
            echo "\tSalary: $player->salary\n";
            echo "\tJob: $player->job\n\n";
        }
    }
}